test/micro/org/openjdk/bench/java/time/format/DateTimeFormatterBench.java

```diff
@@ -118,4 +135,15 @@ public void formatZonedDateTime(final Blackhole blackhole) {
             blackhole.consume(stringBuilder);
         }
     }
+
+    @Benchmark
+    public String formatWithPadding() {
+        return FORMATTER_WITH_PADDING.format(now);
+    }
+
+    @Benchmark
+    public String formatWithZeroPadding() {
+        return YEAR_FORMATTER.format(now);
+    }
+
```

Comment:
There benchmarks are run 4 time each because of the \@Param that are defined for the pre-existing tests.
I don't know if jmh can be annotated to ignore the params; but to avoid wasting resources create a separate jmh source file for thees new tests.

Reply:
Done